packages/manager/cypress/e2e/core/longview/longview.spec.ts

```diff
@@ -266,4 +269,165 @@ describe('longview', () => {
         cy.findByText('Storage').should('be.visible');
       });
   });
+
+  /*
+   * - Tests Longview installation end-to-end using mock API data.
+   * - Confirms that Cloud Manager UI can rename longview client.
+   */
+
+  it('can rename a Longview client on a Linode', () => {
+    const client: LongviewClient = longviewClientFactory.build({
+      api_key: '01AE82DD-6F99-44F6-95781512B64FFBC3',
+      apps: longviewAppsFactory.build(),
+      created: new Date().toISOString(),
+      id: 338283,
+      install_code: '748632FC-E92B-491F-A29D44019039017C',
+      label: randomLabel(),
+      updated: new Date().toISOString(),
+    });
+
+    const newClient: LongviewClient = longviewClientFactory.build({
+      api_key: '01AE82DD-6F99-44F6-95781512B64FFBC3',
+      apps: longviewAppsFactory.build(),
+      created: new Date().toISOString(),
+      id: 338283,
+      install_code: '748632FC-E92B-491F-A29D44019039017C',
+      label: randomLabel(),
+      updated: new Date().toISOString(),
+    });
```

Comment:
```suggestion
    const client: LongviewClient = longviewClientFactory.build();

    const newClient: LongviewClient = longviewClientFactory.build({
    ...client, label: randomLabel()
    });
```

If we just want to create a new (renamed) client with the rest of the fields the same, we could save some space with the spread operator.

Reply:
Updated, thank you!